source/draw/gpu/opengl/backend/gles3/Gles3Context.ooc

```diff
@@ -9,126 +9,20 @@
 version(!gpuOff) {
 use base
 use geometry
-import ../egl/egl
-import ../[GLContext, GLTexture, GLVertexArrayObject, GLIndexBufferObject]
+import ../[GLContext, GLTexture, GLVertexArrayObject, GLIndexBufferObject, EglDisplayContext]
 import external/gles3
 import Gles3Debug, Gles3Fence, Gles3FramebufferObject, Gles3Quad, Gles3Renderer, Gles3ShaderProgram, Gles3Texture, Gles3VolumeTexture, Gles3VertexArrayObject, Gles3IndexBufferObject
 
 Gles3Context: class extends GLContext {
-	_eglContext: Pointer
-	_eglSurface: Pointer
-	_contextCount := static 0
-	_mutex := static Mutex new()
+	displayContext := EglDisplayContext new()
+	getDisplayContextSafely: func -> EglDisplayContext { this == null ? null : this displayContext }
```

Comment:
A bit ugly to compare this to null but the alternatives are to duplicate the code or make the getter static.